src/fileshelter/ui/FileShelterApplication.cpp

```diff
@@ -82,12 +84,16 @@ handlePathChange(Wt::WStackedWidget* stack)
 	{
 		if (wApp->internalPathMatches(index.first))
 		{
+            auto app = dynamic_cast<FileShelterApplication*>(Wt::WApplication::instance());
```

Comment:
You can use FsApp
`FsApp->updateMenuVisibility();`

Reply:
Aha - was searching for it and  passed over #define - of course it's not a comment in C++!